vNext/channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -43,9 +43,12 @@ export class Sender implements IChannelControlsAI {
         throw new Error("Method not implemented.");
     }
 
-    public flush() {
+    public flush(async?: boolean, isBeaconSender?: boolean) {
+        if (isBeaconSender) {
+            this._sender = this._beaconSender;
```

Comment:
This will cause each telemetry item after this flush to be sent with beacons, even if we don't intend to

Reply:
Also if the browser does not support beacons, we will need to ignore this parameter